src/components/new-request/search-components/business-lookup-fetch.vue

```diff
@@ -135,17 +137,21 @@ export default class BusinessLookupFetch extends Mixins(CommonMixin, SearchMixin
     }
 
     if (this.isChangeName) {
-      if (this.isBcBenCccUlc) {
-        this.setLocation(Location.BC)
-        const corpType = this.getSearchBusiness?.legalType as unknown as CorpTypeCd
-        this.setEntityTypeCd(this.corpTypeToEntityType(corpType))
-      } else {
-        this.setEntityTypeCd(this.getSearchBusiness.legalType)
-      }
+      if (this.getSearchBusiness) {
```

Comment:
Check if business selected

Reply:
I don't know if this is similar to the conversion flow, but in that flow if getSearchBusiness is falsy then it executes lines 118-121.

Reply:
I think this okay. if Business look up is cleared and there is no selected business, Company type is still set. This check prevents it. I did test all cases.